.docker/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM jekyll/jekyll:4.0.0
```

Comment:
Should we perhaps drop this to `3.8.6` here as well?

```suggestion
FROM jekyll/jekyll:3.8.6
```

Reply:
It's either that, or wait for our own image and drop that all together.